### DISCUSSION TITLE
Ecom with Magento With NextJs

### DISCUSSION THREAD

**Discussion by iamsharzil** [vercel/next.js#14859] [Show and tell] [answered]
Hey Guys!

I have created a project which uses Magento as a CMS. I want to add this in the example projects of nextJs so that it can be helpful for other developers. It will be helpful if someone guides me.

Here is the link - https://github.com/iamsharzil/cms-magento

**Comment by cristiand391** [comment, marked as answer]
Awesome work!

If you wanna add it in the nextjs examples folder you should check the [contributing guidelines](https://github.com/vercel/next.js/blob/canary/contributing.md#adding-examples).
